File: Array/26_remove-duplicates-from-sorted-array.java
// Problem   : 26. Remove Duplicates from Sorted Array
// Difficulty: Easy
// Link      : https://leetcode.com/problems/remove-duplicates-from-sorted-array/
// Runtime   : 1 ms (beats 77.6%)
// Memory    : 46.7 MB (beats 63.8%)
// Submitted : 2026-08-22 06:19
// Tags      : Array, Two Pointers
class Solution {
    public int removeDuplicates(int[] nums) {
        int k =1;
        for (int j =1; j < nums.length; j++){
            if(nums[j] != nums[j-1]){
                nums[k] =nums[j];
                k = k+1;
            }
        }
        return k;
    }
}

File: Array/26_remove-duplicates-from-sorted-array.cpp
// Problem   : 26. Remove Duplicates from Sorted Array
// Difficulty: Easy
// Link      : https://leetcode.com/problems/remove-duplicates-from-sorted-array/
// Runtime   : 0 ms (beats 100.0%)
// Memory    : 22.7 MB (beats 53.1%)
// Submitted : 2026-08-22 06:20
// Tags      : Array, Two Pointers
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int k = 1;

        for (int j = 1; j < nums.size(); j++) {
            if (nums[j] != nums[j - 1]) {
                nums[k] = nums[j];
                k++;
            }
        }

        return k;
    }
};